x-pack/filebeat/docs/inputs/input-azure-blob-storage.asciidoc

```diff
@@ -97,29 +97,12 @@ processing a certain number of files , decided by the `maximum available workers
         }
     },
     "azure": {
-        "blob": {
+        "storage": {
             "container": {
                 "name": "beatscontainer"
             },
-            "object": {
+            "blob": {
                 "content_type": "application/json",
-                "data": [
```

Comment:
Why this change?

Reply:
This was done in order to make the output follow the same structure as the aws s3 input. After a bit of thought the data field was seeming to be kind of useless as it was just the json parsed version of the data in the message field. I think the message field is sufficient here, it contains the original stringified data.